lib/src/command/dependency_services.dart

```diff
@@ -368,11 +368,16 @@ class DependencyServicesApplyCommand extends PubCommand {
     if (lockFileEditor != null && lockFileEditor.edits.isNotEmpty) {
       writeTextFile(entrypoint.lockFilePath, lockFileEditor.toString());
     }
+    // TODO: Handle HTTP exceptions gracefully!
```

Comment:
I just discovered that this was unhandled, we should probably check how HTTP errors are handled in `dependency-services`.

I suspect things could happen in `acquireDependencies`.